doc/lsst.afw.geom/SpanSet.rst

```diff
@@ -1,6 +1,6 @@
-#######
-SpanSet
-#######
+#########################################################
+Representing regions of pixels with lsst.afw.geom.SpanSet
+#########################################################
 
```

Comment:
Consider "Using lsst.afw.geom.SpanSet to represent irregular regions of pixels"

because it better emphasizes the class name (similar to your change to the title for `Footprint`) and because "irregular" is a useful reminder that a `SpanSet` is probably not rectangular.